docs/sdks/survey-sdk/configuration-fetching.md

```diff
@@ -0,0 +1,78 @@
+# Configuration Fetching
+
+When creating a Survey SDK in Engage Digital you need to specify the **base URL of the bridge** field (see [Getting Started](../quick-start)) which will be used to perform an HTTP request to fetch the survey configuration.
+
+Upon survey creation, Engage Digital will perform a `GET /configuration` call on your bridge to fetch the survey configuration (e.g. if **base URL of the bridge** is `https://survey.bridge.com` then Engage Digital will perform a `GET https://survey.bridge.com/configuration`).
+
+## Response example
+
+Here's an example of a valid Survey SDK configuration:
+``` json
+{
+  "id": "12345",
+  "link": "https://link.tothesurvey.com",
+  "status": "Opened",
+  "questions": [
+    {
+      "id": "main_question",
+      "text": "Are you satisfied with our customer service?",
+      "main": true,
+      "choices": [
+        {
+          "text": "Very satisfied",
+          "value": "5"
+        },
+        ...
+      ]
+    },
+    {
+      "id": "multiple_choices_question",
+      "text": "What can we do to improve our service?",
+      "main": false,
+      "choices": [
+        {
+          "text": "Faster response time"
+        },
+        {
+          "text": "Better follow-up"
+        }
+      ]
+    },
+    {
+      "id": "free_text_question",
+      "text": "Any other suggestion?",
+      "main": false,
+      "choices": []
+    },
+    ...
+  ]
+}
+```
+
+
+### Detailed response fields description
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| id | String | **YES** | Unique identifier of the survey. |
+| link | String | **YES** | Link to access the survey. |
+| status | String | NO | Status of the survey (display only). |
+| questions | Array | **YES** | List of the questions belonging to the survey, see [Question object description](#question-object-description). |
+
+
+### Question object description
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| id | String | **YES** | Unique identifier of the question, the entire question will be ignored if not present,. |
+| text | String | **YES** | Question text (the entire question will be ignored if not present). |
+| main | Boolean | NO | Indicates whether this question is the survey's main question (a survey must have a single main question), will default to `false` if not present. |
+| choices | Array | NO | List of the available choices for the question (can be omitted for free text question), see [Choice object description](#choice-object-description). |
+
+
+### Choice object description
+
+| Field | Type | Mandatory | Description |
+|-|-|-|-|
+| text | String | **YES** | Choice text, the entire choice will be ignored if not present. |
+| value | String | NO | Value of the choice (e.g. "Very satisfied" is equivalent to 5/5). |
```

Comment:
there's a mention to that in the responses polling part of the documentation but it may not be explicit enough

Reply:
Good point, please be more explicit.

Also the example is a bit weird. I would expect that `e.g. "Very satisfied"...` means that I can put "Very satisfied" in this value field, but this value field is for `"5"`, isn't it?

Reply:
AFAIR there was no strong requirement on this so I went with how it's done in Alchemer, i.e. the response value needs to match the choice's value so that it is mapped to the choice text but not the other way around

Reply:
1. So if there is no value here in the configuration, and the text is "Very satisfied", and on the response polling, we need to have the value to "Very satisfied" too, right? If so, that's what needs to be more explicit
2. Can we just change the examples to something like:
```suggestion
| value | String | NO | Value of the choice (e.g. "5", which is equivalent to a "Very satisfied" text value). |
```

Reply:
thanks, suggestion applied